imblearn/utils/tests/test_validation.py

```diff
@@ -315,3 +316,43 @@ def test_sampling_strategy_check_order(
         sampling_strategy, y, sampling_type
     )
     assert sampling_strategy_ == expected_result
+
+
+def test_output_formater_plain_list():
```

Comment:
Add a test that we pass a numpy array as well?

Reply:
I will add.